_posts/2017-02-14-whats-in-7.0.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "What's in 7.0"
+author: Henry Zhu
+date:   2017-02-14 12:00:00
+categories: announcements
+share_text: "What's in 7.0"
+---
+
+If you didn't know already, we're planning on releasing a 7.0 version soon! I'd like to go over some notable changes and a few things we are still working out!
+
+> For more specific changes, please check out our upgrade guide (will be updated)!
+
+## 💥 Breaking Changes
+
+### [#4315](https://github.com/babel/babel/issues/4315) Drop support for unmaintained Node versions: 0.10, 0.12, 5
+
+When OSS maintainers want to move the project forward and make breaking changes, it may come at the cost of more hassle for their users.
+
+A major version bump *is* just a number, but it can cause unnecessary pain for users/library authors (including ourselves since we'd want to upgrade Babel itself to run on Babel 7 at some point right). We didn't want to do it just for this change.
```

Comment:
What about:

> Progress and moving forward in OSS projects often come with the cost of hassle and pain for its users. Because of this, we're always been hesitant in making the choice to introduce a major version bump/breaking changes. But by dropping Node v0.12, we can not only make a number of improvements to the codebase, but also upgrade dependencies and tools (ESLint, Yarn, Jest, Lerna, etc).